testing/simulator/src/checks.rs

```diff
@@ -172,15 +173,17 @@ pub async fn verify_fork_version<E: EthSpec>(
 /// have full aggregates.
 pub async fn verify_full_sync_aggregates_up_to<E: EthSpec>(
     network: LocalNetwork<E>,
+    db: Arc<HotColdDB<E, LevelDB<E>, LevelDB<E>>>,
     sync_committee_start_slot: Slot,
     upto_slot: Slot,
     slot_duration: Duration,
 ) -> Result<(), String> {
     slot_delay(upto_slot, slot_duration).await;
-    let remote_nodes = network.remote_nodes()?;
-    let remote_node = remote_nodes.first().unwrap();
+    let anchor_info = db.get_anchor_info();
+    let state_upper_limit = anchor_info.state_upper_limit;
+    let effective_start_slot = max(sync_committee_start_slot, state_upper_limit);
```

Comment:
This is just a test, to change the functionality I think you need to modify `sync_committee_duties` in `beacon_node/http_api/src/sync_committees.rs`. Especially the function `duties_from_state_load` and the variable `load_slot`.